.travis.yml

```diff
@@ -1,37 +1,82 @@
 language: php
 
-php:
-  - 7.2
-  - 7.3
-  - 7.4
-
-sudo: false
-
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.2
-      env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: 7.4
-      env: SYMFONY_VERSION=^4.4
-    - php: 7.4
-      env: SYMFONY_VERSION=^5.0
+os:
+  - linux
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - vendor
 
 before_install:
+  - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
 
 install:
   - travis_retry composer update --prefer-dist --no-interaction --no-suggest $COMPOSER_FLAGS
 
 script:
-  - mkdir -p build/logs
-  - ./vendor/bin/phpunit --coverage-clover build/logs/clover.xml
-  - ./vendor/bin/phpcs --standard=php_cs.xml --ignore=*/vendor/* .
```

Comment:
Err, where did the code style go? 🤔 

Did you replace it with a run of php-cs-fixer? If that's the case, I have two concerns about this approach:

1) php-cs-fixer is not a linter tool, so this solution doesn't allow running the code style linter in the IDE to display violations
2) I've recently added a code style rule to enforce constant visibility and that check is gone now, since php-cs-fixer only cares about formatting issues (maybe psalm has a rule for this?)

Reply:
OK so I'll keep it. I was not sure for that change.